admin/src/pages/UserSearch.spec.js

```diff
@@ -101,8 +101,8 @@ describe('UserSearch', () => {
               searchText: '',
               currentPage: 1,
               pageSize: 25,
-              notActivated: true,
-              isDeleted: null,
+              filterByActivated: false,
+              filterByDeleted: null,
```

Comment:
Same why null instead of false?
```suggestion
              filterByDeleted: false,
```

Reply:
Because we switsch between `null` and `true`.